amenity-points.mss

```diff
@@ -1557,12 +1590,11 @@
 
   // waste_disposal tagging on ways - tagging on nodes is defined later
   [feature = 'amenity_waste_disposal'][zoom >= 19] {
-    [access = null],
-    [access = 'permissive'],
-    [access = 'yes'] {
-      marker-file: url('symbols/amenity/waste_disposal.svg');
-      marker-fill: @amenity-brown;
-      marker-placement: interior;
+    marker-file: url('symbols/amenity/waste_disposal.svg');
+    marker-fill: @amenity-brown;
+    marker-placement: interior;
+    [access != ''][access != 'permissive'][access != 'yes'] {
+      marker-opacity: 0.33;
```

Comment:
rendering private waste disposal seems bizarre to me

Reply:
I removed the rendering of private `waste_disposal`